### PR TITLE
feat: update VFolder operation statuses

### PR DESCRIPTION
**Pull request by rapsealk**
This PR is a follow-up of lablup/backend.ai#835.

This PR adds a type to represent VFolder's operations statuses and updated some code as the upstream PR has renamed VFolder status `deleting` to `delete-ongoing`.

Minimum required manager version: 24.03

**Checklist:** (if applicable)

- [x] Mention to the original issue
- [ ] Documentation
- [x] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Review by agatha197 — APPROVED** [review]
LGTM

File: src/components/backend-ai-storage-list.ts [hunk 1]
```diff
@@ -1619,8 +1629,16 @@ export default class BackendAiStorageList extends BackendAIPage {
     });
   }
 
-  _checkProcessingStatus(status) {
-    return ['performing', 'cloning', 'deleting', 'mounted'].includes(status);
+  _isUncontrollableStatus(status: VFolderOperationStatus) {
+    return [
+      'performing',
+      'cloning',
+      'mounted',
```
Comment (yomybaby) [inline src/components/backend-ai-storage-list.ts:1621]: This question is not about the changes in this PR. Is it correct that the `mounted` status is an uncontrollable status and disables the share feature?
@rapsealk  @agatha197

Comment (rapsealk) [reply src/components/backend-ai-storage-list.ts:1621]: In accordance with the Backend.AI manager implementation, the status `VFolderOperationStatus.MOUNTED` is interpreted as both `READABLE` and `UPDATABLE`, signifying that a mounted status is still eligible for sharing. The relevant code snippet can be found at:
https://github.com/lablup/backend.ai/blob/32a0553839b489b83daa226a90ad482cda87c788/src/ai/backend/manager/api/vfolder.py#L147-L160

**Comment by rapsealk** [comment]
@agatha197 Resolved conflicts by deleting `session-launcher-neo.ts`. Please check!

**Pull request merged**